data/mods/Xedra_Evolved/snippets/e_logs.json

```diff
@@ -6,41 +6,67 @@
       }
     ]
+  },
+  {
+    "type": "snippet",
+    "category": "local_files_simple",
+    "//": "This is extending the vanilla snippets belonging to mostly work of random people: chat logs, SMSes, popular photos and videos that people share, or something they can film by themselves.",
+    "text": [
+      {
+        "id": "xe_local_files_simple_1",
+        "name": "cryptid sighting werewolf",
+        "text": "hunternetvulfgar.mp4\nThe video cuts in as a montage of school security cameras in hallways as a young man and woman break into the school.  As they walk past each camera that image drops and the others rearange larger while a popular song from the 90's begins playing.  The young adults enter a classroom and are clearly flirting.\nAwoo oooh, got you where I want you\nplays over top of the video.  The man laughs at something the woman says and throws his head back laughing while she is clearly annoyed.  His face elongates as his body grows shredding the clothes he is wearing as limbs change and sprout thick fur.  She has backed away from him and put the students and teacher's desks between them.  Fully transformed into a hideous wolfman he begins his approach towards his once date now prey.  Smashing chairs and students desks out of his way.  Suddenly the wolfcreature pauses and ears prick up.  It turns and looks at the wall of windows in the classroom.  Glass erupts into the room as the darkness outside the window shows the muzzle flare of at least a dozen different guns firing.  The wolfcreature is hit more times than you can count in a span of seconds and falls to the ground.\nThe song reaches a crescendo 'Got you where I want you, got you, yeahhhhh!'\nAs his date pulls a large caliber pistol out of the teachers desk, walks over to the fallen beast and calmly puts three bullets in its skull.  The video ends."
```

Comment:
```suggestion
        "text": "hunternetvulfgar.mp4\nThe video cuts in as a montage of school security cameras in hallways, as a young man and woman break into the school.  As they walk past each camera, that image drops and the others rearrange larger, while a popular song from the 90's begins playing.  The young adults enter a classroom and are clearly flirting.\nAwoo oooh, got you where I want you\nplays over top of the video.  The man laughs at something the woman says, and throws his head back laughing while she is clearly annoyed.  His face elongates as his body grows, shredding the clothes he is wearing as limbs change and sprout thick fur.  She has backed away from him and put the students' and teacher's desks between them.  Fully transformed into a hideous wolfman, he begins his approach towards his once date now prey, smashing chairs and students' desks out of his way.  Suddenly the wolfcreature pauses and its ears prick up.  It turns and looks at the wall of windows in the classroom.  Glass erupts into the room as the darkness outside the window shows the muzzle flare of at least a dozen different guns firing.  The wolfcreature is hit more times than you can count in a span of seconds and falls to the ground.\nThe song reaches a crescendo 'Got you where I want you, got you, yeahhhhh!' as his date pulls a large caliber pistol out of the teachers desk, walks over to the fallen beast and calmly puts three bullets in its skull.  The video ends."
```
Added some commas to make the sentece structure read clearer and few other minor readability improvements